content/docs/04-clusters/2-nested-clusters/nested-cluster-quickstart.md

```diff
@@ -19,11 +19,11 @@ With the Palette Nested Cluster option, you can deploy additional Kubernetes clu
 
 There are certain advantages to use a nested cluster environment. For example, you are free to operate with admin level privileges, while simultaneously ensuring strong isolation, reducing operational overhead, and improving resource utilization.
 
-Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within the Palette orchestrator.
+Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster) and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within Palette.
```

Comment:
@Siboney1 , here enable is for the host cluster not for the nested cluster

Reply:
This sentence refers to the steps Enabling a Nested Cluster as the document is about actions with Nested Clusters. I am describing steps to create a Nested Cluster. Steps to create a Host Cluster is described under Prerequisites.

Reply:
This step is for enabling a host cluster. The link gives them the instructions.

Reply:
then frame it as :
Try the steps below to [enable](/clusters/nested-clusters/cluster-quickstart#enablinganestedclusteronahostcluster)  the host cluster and [deploy](/clusters/nested-clusters/cluster-quickstart#deployinganestedcluster) a managed Nested Cluster from within Palette.